pyproject.toml

```diff
@@ -23,7 +23,7 @@ pydantic = "1.10.14"
 networkx = "^3.1"
 openai = "^1.0rc"
 rich_click = "^1.6.1"
-tiktoken = "^0.4.0"
+tiktoken = ">0.4.0"
```

Comment:
**Suggestion:** Specifying the `tiktoken` dependency with a lower bound version only (`>0.4.0`) might lead to unexpected compatibility issues in the future if `tiktoken` releases a version with breaking changes. It's recommended to use a version range that caps the maximum version to avoid such issues, especially for libraries that are under active development and might introduce breaking changes in minor or major version updates. [best practice]
```suggestion
tiktoken = ">=0.4.0,<1.0.0"
```